modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/executors/LCExecutor.java

```diff
@@ -65,6 +65,9 @@ public boolean execute(RequestContext context, String currentState, String targe
                     setOtherRolePermission(role, currentState, targetState);
                 }
             }
+            if (currentState.equals("In-Review") && targetState.equals("Published")) {
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7